pkg/cnab/config-adapter/stamp.go

```diff
@@ -152,17 +173,21 @@ func LoadStamp(bun cnab.ExtendedBundle) (Stamp, error) {
 	return stamp, nil
 }
 
-// getUsedMixinsVersion compare the mixins defined in the manifest and the ones installed and then retrieve the mixin's version info
-func (c *ManifestConverter) getUsedMixinsVersion() map[string]string {
-	usedMixinsVersion := make(map[string]string)
+// getUsedMixinRecords compare the mixins defined in the manifest and the ones installed and then retrieve the mixin's version info
```

Comment:
nit: Update comment now that we're returning the record instead of just version info